src/packages/ganache/webpack/webpack.cli.config.ts

```diff
@@ -9,7 +9,21 @@ const config: webpack.Configuration = merge({}, base, {
   output: {
     path: path.resolve(__dirname, "../", "dist", "cli")
   },
-  externals: ["bigint-buffer", "leveldown", "secp256k1", "keccak"],
+  externals: [
+    "bigint-buffer",
+    "leveldown",
+    "secp256k1",
+    "keccak",
+    {
+      "@ganache/core": path.resolve(
+        __dirname,
+        "../",
+        "dist",
+        "node",
+        "ganache.min.js"
+      )
+    }
```

Comment:
I think this will need to be something like `@ganache/core": path.join("../", "node", "ganache.min.js")`, otherwise your absolute path with be what is used to resolve `@ganache/core` in the bundle.

Reply:
Hmm isn't `resolve` ok? It's basically like calling `cd __dirname` then the rest of the args are the constructed path you want to go to from that initial `cd`.

The output path above uses `resolve` as well.

Reply:
Doing this writes _your_ path into the final bundle. So it will work on your machine, in that specific location, but not in another other machine or location.

Reply:
Cool, looks like that works